### ISSUE TITLE
token being drag instead of teleported

### ISSUE THREAD

**Issue by masterlems** [grandseiken/foundryvtt-multilevel-tokens#98]
when a token enter a teleportation spot the token is being drag to the new destination instead of being teleported, the token/player can see everything that the token go through.

on foundry vtt 8.6

**Comment by vanderleitelesjr** [comment]
same problem here, even if the Animate Movement option is checked out, the token still animates when being teleported within the same scene.

**Comment by DarkPhonix5097** [comment]
Yes  I have the same Problem and the whole map was discoverd when you use fog of war.

**Comment by grandseiken** [comment]
I don't experience this issue, do you have any other modules enabled?

**Comment by vanderleitelesjr** [comment]
It turns out to be an incompatibility with Token Attacher

**Comment by masterlems** [comment]
sorry for the delay of response i was at work. i tried with token attacher turn off and everything was ok so yes it a compatibility problem between these two.
